workers/unity/Packages/com.improbable.gdk.core/Authority/AuthorityComponents.cs

```diff
@@ -2,14 +2,27 @@
 
 namespace Improbable.Gdk.Core
 {
+    /// <summary>
+    ///     This ECS component denotes that this worker is authoritative over T.
```

Comment:
For `over T.`, either `over <see cref="T"/>.`, or `over the SpatialOS component.`